src/index.js

```diff
@@ -279,6 +275,11 @@ export default class Carousel extends React.Component {
               )
             );
 
+        if (length >= 10) {
+          if (this.clickDisabled === false) this.props.onDragStart(e);
+          this.clickDisabled = true;
```

Comment:
Probably should have disabled click here anyway, which prevents unwanted click events from firing during swiping on touch devices!